apps/condo/domains/user/schema/ForgotPasswordAction.test.js

```diff
@@ -0,0 +1,162 @@
+            const result = await client.mutate(START_PASSWORD_RECOVERY_MUTATION, {
+                data: { email: userAttrs.email, dv: 1, sender: { dv: 1, fingerprint: 'tests' } },
+            })
+            expect(result.data.result).toEqual({ status: 'ok' })
+        })
+
+        it('can change password', async () => {
+            const admin = await makeLoggedInAdminClient()
+            const [user, userAttrs] = await createTestUser(admin)
+            const client = await makeClient()
+
+            const [{ token }] = await createTestForgotPasswordAction(admin, user)
+            const password = `new_${userAttrs.password}`
+            const result = await client.mutate(CHANGE_PASSWORD_WITH_TOKEN_MUTATION, { data: { token, password } })
+            expect(result.data.result).toEqual({ status: 'ok' })
+
+            const newClient = await makeLoggedInClient({ email: userAttrs.email, password })
+            expect(newClient.user.id).toEqual(user.id)
+        })
+
+        it('cant change password with expired token', async () => {
+            const admin = await makeLoggedInAdminClient()
+            const [user, userAttrs] = await createTestUser(admin)
+            const client = await makeClient()
+
+            const [{ token, id }] = await createTestForgotPasswordAction(admin, user)
+            const password = `new_${userAttrs.password}`
+
+            await updateTestForgotPasswordAction(admin, id, { expiresAt: new Date(Date.now()).toISOString() })
+            const result = await client.mutate(CHANGE_PASSWORD_WITH_TOKEN_MUTATION, { data: { token, password } })
+            expect(result.errors).toHaveLength(1)
+            expect(result.errors[0].message).toEqual(`${RESET_TOKEN_NOT_FOUND}] Unable to find token`)
+        })
+
+        it('cant change password to a shorter one', async () => {
+            const admin = await makeLoggedInAdminClient()
+            const [user, userAttrs] = await createTestUser(admin)
+            const client = await makeClient()
+
+            const [{ token }] = await createTestForgotPasswordAction(admin, user)
+            const password = userAttrs.password.slice(0, MIN_PASSWORD_LENGTH - 1)
+            const result = await client.mutate(CHANGE_PASSWORD_WITH_TOKEN_MUTATION, { data: { token, password } })
+            expect(result.errors).toHaveLength(1)
+            expect(result.errors[0].message).toEqual(`${PASSWORD_TOO_SHORT}] Password too short`)
+        })
+
+        it('can check token expired status', async () => {
+            const admin = await makeLoggedInAdminClient()
+            const [user] = await createTestUser(admin)
+            const client = await makeClient()
+
+            const [{ token }] = await createTestForgotPasswordAction(admin, user)
+            const result = await client.mutate(CHECK_PASSWORD_RECOVERY_TOKEN, { data: { token } })
+            expect(result.data.result).toEqual({ status: 'ok' })
+        })
+    })
+})
```

Comment:
What about others tests?
 - `start recovery for unknown email`
 - `change password to empty`
 - `reset forgotten password`

Why did you just delete the existing tests?

Why you don't have any test for a non-anonimous user?